addon/utils/model-factory.js

```diff
@@ -71,21 +84,19 @@ export function getTopLevelPropertyGroups({ store, graphs, form }) {
   if (
     getFormModelVersion(form, {
       store,
-      formGraph: graphs.formGraph,
-    }).toLowerCase() == 'v2'
+      formGraph: formGraph,
+    }).toLowerCase() == V2
```

Comment:
changed it mostly as a reflex because coming from ts, you'd get some automatic shouting if you tried to use an import that doesn't exist... no such luck in javascript of course...  Still feels a bit magic constant to repeat a string throughout the code no?

Not sure about the history of this file either but a refactor a big as this would probably warrant it's own PR?

Reply:
> Still feels a bit magic constant to repeat a string throughout the code no?

For sure, I just don't think that moving something into a constant was the "fix" for this magic string here. It _feels_ worse to use a constant and then having to know what it contains instead of just using the magic string. If for some reason the constant would change, this code would need to be updated as well. I probably would have made a function that did the check instead, but that seems to exist in the helper package already (but not public).

In any case, it's not worth changing it back either, so just ignore this remark. Sorry for the noise.

> Not sure about the history of this file either but a refactor a big as this would probably warrant it's own PR?

Yea, I was just thinking out loud about what the "next steps" could be.